### ISSUE TITLE
Permissions

### ISSUE THREAD

**Issue by MichaelVessia** [adelphimathcs/ProjectEuler-TEST#1]
This repo shouldn't be on this team.  See https://github.com/adelphimathcs/repoBuilder/issues/6

**Comment by b-stone** [comment]
From what I can tell this is a teamless repo. Also I can push to it which is to be expected.

**Comment by MichaelVessia** [comment]
@adelphimathcs/alumni can you see this repo?

**Comment by MattMatero** [comment]
Yes, I can see this repo. I also have one for ProjectEuler-alumniTest which was made w/ the script passing the alumni ID. @b-stone does that one seem teamless as well?
